scripts/qa_crowbarsetup.sh

```diff
@@ -1273,10 +1247,7 @@ EOF
 
     if iscloudver 6plus ; then
         onadmin_prepare_sles12_repos
-        if [ -n "$want_sles12sp1" ] ; then
-            onadmin_prepare_sles12sp1_repos
-        fi
-        onadmin_prepare_sles12plus_cloud_repos
```

Comment:
I'm not sure that removing that last line is fully correct, see what onadmin_create_sles12plus_repos does.